docs/disconnected-cincinnati-operator.md

```diff
@@ -240,6 +240,9 @@ You might want to review the documentation around disconnected registries to lea
 
     gives a full URI.
 
+    If you see `ReconcileCompleted` status as `false` with reason `CreateRouteFailed` caused by `host must conform to DNS 1123 naming convention`
+    and `must be no more than 63 characters`, try creating the Update Service with a shorter name. The resource name should be 63 characters or less. 
```

Comment:
> The resource name should be 63 characters or less.

"The resource" is a bit ambiguous (UpdateService?  Route?).  And really, it's the combination of the Route's name and namespace.  Can we rely on the quoted `must be no more than 63 characters` to address @LalatenduMohanty's [request for the 63-char mention][1], rely on the rest of the Route-tooling error message for further context, and drop the "The resource..." sentence?

Also nit:

```console
$ git --no-pager show --oneline --check origin/pr/96
070a7a0 (origin/pr/96) document using shorter name for cincinnati cr
docs/disconnected-cincinnati-operator.md:244: trailing whitespace.
+    and `must be no more than 63 characters`, try creating the Update Service with a shorter name. The resource name should be 63 characters or less. 
docs/disconnected-cincinnati-operator.md:245: trailing whitespace.
+    
```

[1]: https://github.com/openshift/cincinnati-operator/pull/96#discussion_r562803709

Reply:
I do not see any reason to drop the sentence because this is in the documentation and there is no harm if we add little more information.

Reply:
> I do not see any reason to drop the sentence...

I don't think it adds anything to the `host must conform to DNS 1123 naming convention` and `must be no more than 63 characters` quoted in the previous sentence.  But whatever, if we keep it, can we at least clarify "The resource" to "The UpdateService" or whatever it was intended to be?

Reply:
Removed "The resource" line, as it would have caused more confusion than it was solving.